tests/utils.go

```diff
@@ -757,6 +758,10 @@ func BeforeTestSuitSetup() {
 	CreatePVC(osWindows, defaultWindowsDiskSize, Config.StorageClassWindows)
 	CreatePVC(osRhel, defaultRhelDiskSize, Config.StorageClassRhel)
 
+	if IsRunningOnKindInfraIPv6() {
```

Comment:
Why not use `IsRunningOnKindInfra` ? Isn't the issue here related to kind itself ?

Reply:
i added to kubevirtci a need part that generates the files only for ipv6, this is the reason
in the future if we add it to all the other kind providers (i.e sriov)
this can be changed
mounting when file doesnt exists will break virt launcher

Reply:
moreover the change need to be both on kci and here, as groomed before, because it was decided by Petr / Alona and even you at the first that kubevirt shouldnt be changed, but the change is part of the test
(kci cant do it alone, because it depends on the namespace name, which is decided, created and deleted in the tests themself)